cmd/app/options/ai.go

```diff
@@ -0,0 +1,55 @@
+package options
```

Comment:
You can execute `make fix-license` to batch supplement license comment in the project root directory.